components/common/shoppingCart/utils.ts

```diff
@@ -84,3 +84,19 @@ export const nftToListingCartItem = (
     token: nft.token,
   }
 }
+
+export const shoppingCartItemToListingCartItem = (
+  item: ShoppingCartItem,
+  floor = '',
+): ListCartItem => {
+  return {
+    id: item.id,
+    name: item.name,
+    price: item.price ?? '0',
+    urlPrefix: item.urlPrefix,
+    collection: { ...item.collection, floor },
+    metadata: item.metadata,
+    meta: item.meta,
+    listPrice: null,
```

Comment:
Type 'null' is not assignable to type 'number'.

Reply:
fixed

Reply:
fixed